router/relay/relay.go

```diff
@@ -918,7 +918,7 @@ func (rst *RelayStateImpl) DeployPrepStmt(qname string) (shard.PreparedStatement
 		Str("query", query).
 		Uint64("hash", hash).
 		Str("client", rst.Client().ID()).
-		Str("shard", rst.Client().Server().Datashards()[0].ID()).
+		Strs("shards", rst.Client().Server().DatashardIds()).
```

Comment:
```suggestion
		Strs("shards", rst.Client().Server().DatashardIds()[0]).
```

Reply:
panic: runtime error: index out of range